docs/source/using-discovery/usingslurm.rst

```diff
@@ -59,42 +103,376 @@ To run a job interactively, use ``srun``. To submit a job to run in the backgrou
+   8                   56                  ib,cascadelake,next d[0001-0008]
+   120                 56                  ib,cascadelake,prod d[0009-0128]
+   32                  20                  ivybridge,prod      c[3000-3006,3008-303
+   115                 24                  lenovo,rapl,haswell,c[0156,0158-0159,016
+   381                 28                  broadwell,prod      c[0336-0343,0376-040
+   4                   16                  sandybridge,largememc[2000-2003]
+   2                   112                 cascadelake,ib,prod d[0129-0130]
+   20                  128                 zen2,ib,prod        d[0131-0150]
+View what nodes are in what state in a partition using ``statecompact``::
+
+   sinfo -p lopez --Format=time,nodes,statecompact,features,memory,cpus,nodelist
+Using scontrol
+--------------
+The ``scontrol`` command is used for monitoring and modifying queued, running jobs, and reservations.
+
+Example uses
+^^^^^^^^^^^^
+View information about a specific node::
+
+   scontrol show node -d <node_name>
+For example::
+
+   $ scontrol show node -d c2180
+   NodeName=c2180 Arch=x86_64 CoresPerSocket=14
+   CPUAlloc=0 CPUTot=28 CPULoad=0.01
+   AvailableFeatures=broadwell,prod
+   ActiveFeatures=broadwell,prod
+   Gres=gpu:k80:7(S:0-1)
+   GresDrain=N/A
+   GresUsed=gpu:k80:0(IDX:N/A)
+   NodeAddr=c2180 NodeHostName=c2180 Version=21.08.8-2
+   OS=Linux 3.10.0-1160.25.1.el7.x86_64 #1 SMP Wed Apr 28 21:49:45 UTC 2021
+   RealMemory=512000 AllocMem=0 FreeMem=486591 Sockets=2 Boards=1
+   State=IDLE ThreadsPerCore=1 TmpDisk=0 Weight=6 Owner=N/A MCS_label=N/A
+   Partitions=gpu,multigpu,reservation
+   BootTime=2022-12-14T07:23:47 SlurmdStartTime=2022-12-23T07:40:56
+   LastBusyTime=2023-01-19T14:40:02
+   CfgTRES=cpu=28,mem=500G,billing=728,gres/gpu=7
+   AllocTRES=
+   CapWatts=n/a
+   CurrentWatts=0 AveWatts=0
+   ExtSensorsJoules=n/s ExtSensorsWatts=0 ExtSensorsTemp=n/s
+For information on all reservations, this command will show information about a specific node in the cluster, including the node name, state, number of CPUs, and amount of memory::
+
+   scontrol show reservations
+
+View information about a specific job,this command will show information about a specific job, including the job ID, state, user name, and partition name. ::
+
+   scontrol show job <job_id>
+
+View information about a specific reservation, this command will show information about a specific reservation in the cluster, including the reservation name, start time, end time, and nodes included in the reservation::
+
+   scontrol show reservation <reservation_name>
+
+These are just a few examples of what you can do with ``sinfo`` and ``scontrol`` to view information about the state of the cluster and specific nodes. There are many other options and commands available, and it is recommended to consult the `Slurm documentation`_ for more information on how to use these tools effectively.
+
+
+Best practices
```

Comment:
Can we format this section a little differently, like either bold the main point before the colon in bold, or drop the info after the colon to the next line? Just to break up the section a bit, maybe even add a line between points.

Reply:
I do not entirely understand this comment. I added an extra line between items in the numbered list. Is that what you meant?